src/Codegen/Generator.hack

```diff
@@ -267,6 +272,31 @@ class QueryField extends Field {
 
 class MutationField extends QueryField {}
 
+class EnumType implements GeneratableObjectType {
+    public function __construct(
+        private DefinitionFinder\ScannedEnum $scanned_enum,
+        private \Slack\GraphQL\EnumType $enum_type,
+       ) {}
+
+    public function generateObjectType(HackCodegenFactory $cg): CodegenClass {
+        return $cg->codegenClass($this->enum_type->getType())
```

Comment:
This is quite similar to `generateObjectType` for `CompositeType`; maybe we can dry this up later.